blog/20240124-simplex-chat-infrastructure-costs-v5-5-simplex-ux-private-notes-group-history.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: layouts/article.html
+title: "SimpleX Chat: free infrastructure from Linode, v5.5 released with a simpler UX to connect, private notes and group history"
+date: 2024-01-24
+# previewBody: blog_previews/20231125.html
+# image: images/20231125-mobile2.png
+draft: true
+permalink: "/blog/20240124-simplex-chat-infrastructure-costs-v5-5-simplex-ux-private-notes-group-history.html"
+---
+
+# SimpleX Chat: free infrastructure from Linode, v5.5 released with a simpler UX to connect, private notes and group history.
+
+**Published:** Jan 24, 2024
+
+[SimpleX Chat infrastructure on Linode](#simplex-chat-infrastructure-on-linode)):
+- Free infrastructure.
+- SimpleX servers in Linode Marketplace.
+- High capacity messaging servers.
+
+What's new in v5.5:
+- [private notes](#private-notes)
+- [group history](#group-history)
+- [simpler UX to connect to other users](#simpler-ux-to-connect-to-other-users)
+- [message delivery stability and other improvements](#message-delivery-stability-and-other-improvements)
+
+Also, we added Hungarian (only Android) and Turkish interface languages, thanks to [the users' community and Weblate](https://github.com/simplex-chat/simplex-chat#help-translating-simplex-chat).
+
+SimpleX Chat Android app is now available in 20 languages!
+
+## SimpleX Chat infrastructure on Linode
+
+We chose Linode as our hosting provider because prior to founding SimpleX Chat I used it for more than a decade, and they have been consistently reliable, cheaper than alternatives, with excellent support and great documentation.
+
+When Linode was acquired by Akamai, we were a bit nervous about how it may affect service quality. So far it's been working out quite well.
+
+As the usage of SimpleX network was growing, so did our hosting costs, and from being really small they started to become significant, particularly as we didn't yet manage to optimise the servers last year.
+
+Linode came to the rescue - we're really excited to announce that Akamai decided to support SimpleX Chat growth by accepting it into their [Linode Rise startup programme](https://www.linode.com/linode-for-startups/).
+
+Thanks to this programme, we received:
```

Comment:
```suggestion
Thanks to this program, we received:
```

Reply:
only correctly links with the first point, better to include "we received" into it. made separate suggestion

Reply:
or maybe even don't make it a list, since second point even says "This is unrelated to the Rise program"